client/web/src/components/series/list/SeriesList.tsx

```diff
@@ -13,44 +13,39 @@ import {
 } from '@material-ui/core'
 import SyncIcon from '@material-ui/icons/Sync'
 import { useTranslation } from 'react-i18next'
-
 import { useDispatch, useSelector } from 'react-redux'
 
 import { actionTypes, selectors } from '../../../features/series'
 import styles from './SeriesList.style'
+import { HttpRequest } from '../../../features/common/HttpRequest'
 
 const useStyles = makeStyles(styles)
 
 type SeriesListProps = {}
 
 export const SeriesList: React.FC<SeriesListProps> = () => {
   const { list } = useSelector(selectors.getSeriesStore)
-
-  const classes = useStyles()
   const { t } = useTranslation()
+  const classes = useStyles()
+  const dispatch = useDispatch()
 
-  const updateSeriesList = () => {
-    axios.get('/api/series').then((response) => {
-      setList(response.data)
-    })
-  }
+  useEffect(() => {
+    dispatch({ type: actionTypes.FETCH_ALL_SERIES })
+  }, [dispatch])
 
-  const refresh = () => {
-    axios.get('/api/PacsMirror').then(() => {
-      updateSeriesList();
+  const synchronize = () => {
+    let httpRequest = new HttpRequest({ baseURL: '/api' })
```

Comment:
Bad place, we should move the code :
- First: add new Feature folder Pacs
- first: create one service about Pacs  with your HttpRequest
- Next: call the method into serieSaga => fetchSeries using code below:
```typescript
yield call([pacService, 'fetchMiror'])
....
```

For more information : https://redux-saga.js.org/docs/advanced/SequencingSagas.html
the instruction yield is waiting for the function is terminated to continue the process

Doing with that method, we remove business logic from components